web-registry/src/features/marketplace/BuySellOrderFlow/BuySellOrderFlow.tsx

```diff
@@ -191,13 +191,13 @@ export const BuySellOrderFlow = ({
    * ui update effect
    */
   useEffect(() => {
-    if (projects && isFlowStarted && accountAddress) {
```

Comment:
We had an infinite loop here because projects are given as arrays above and the ref was changing on each render.